js/modules/k6/k6.go

```diff
@@ -43,14 +45,17 @@ var ErrGroupInInitContext = common.NewInitContextError("Using group() in the ini
 // ErrCheckInInitContext is returned when check() are using in the init context
 var ErrCheckInInitContext = common.NewInitContextError("Using check() in the init context is not supported")
 
+// New returns a new module Struct
 func New() *K6 {
 	return &K6{}
 }
 
+// Fail is a fancy way of saying `throw "something"`
 func (*K6) Fail(msg string) (goja.Value, error) {
 	return goja.Undefined(), errors.New(msg)
 }
 
+// Sleep waits for the provides seconds before continuing the execution
```

Comment:
```suggestion
// Sleep waits the provided seconds before continuing the execution.
```

Sidenote: shouldn't all these comments end in period?

Reply:
:shrug: 

I literally wanted to remove all linter errors and wrote something just to remove it :P

Reply:
apparently .. .yes https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences